### PR TITLE
add docs for NATS-Jestream connector

### PR DESCRIPTION
**Pull request by neha-Gupta1**
add tutorial to demonstrate how to use a NATS Jetstream trigger to invoke a function. 
it includes-
- step NATS server
- creating functions 
- testing the result

### CODE REVIEW

File: content/en/docs/usage/triggers/message-queue-trigger-kind-keda/nats-jetstream.md [hunk 1]
```diff
@@ -0,0 +1,172 @@
+---
+title: "NATS Jetstream"
+draft: false
+description: "Keda based Message Queue Trigger for NATS Jetstream"
+weight: 5
+---
+
+This tutorial will demonstrate how to use a NATS Jetstream trigger to invoke a function.
+We'll assume you have Fission and Kubernetes installed.
+If not, please head over to the [install guide]({{% ref "../../../installation/_index.en.md" %}}).
```
[the remark this answers is not in the document]
Comment (neha-Gupta1) [reply content/en/docs/usage/triggers/message-queue-trigger-kind-keda/nats-jetstream.md:10]: it is a link

File: content/en/docs/usage/triggers/message-queue-trigger-kind-keda/nats-jetstream.md [hunk 2]
```diff
@@ -0,0 +1,172 @@
...
+## Installation
+
+If you want to setup NATS Jetstream server on the Kubernetes cluster, you can use the [information here](https://github.com/nats-io/k8s) or you can check the documentation for nats jetstream [docs](https://docs.nats.io/running-a-nats-service/nats-kubernetes).  
+You can also setup NATS jetstream server with this [yaml](https://github.com/fission/keda-connectors/blob/master/nats-jetstream-http-connector/test/jetstream/jetstream-server.yaml) file.(Monitoring is already configured)
+
+{{% notice info %}}
+NATS jetstream keda connector uses NATS monitoring to scale the deployment, to enable monitoring in nats we need to pass flags as below, you can get more [information here](https://docs.nats.io/nats-server/configuration/monitoring)
+
+```bash
+-m, --http_port PORT             HTTP PORT for monitoring
```
[the remark this answers is not in the document]
Comment (neha-Gupta1) [reply content/en/docs/usage/triggers/message-queue-trigger-kind-keda/nats-jetstream.md:23]: bash command

**Comment by sanketsudake** [comment]
@mergifyio rebase

**Comment by sanketsudake** [comment]
LGTM

**Pull request merged**